src/simsopt/field/tracing.py

```diff
@@ -167,7 +170,7 @@ def trace_particles(field: MagneticField, xyz_inits: NDArray[Float],
         if not forget_exact_path:
             res_tys.append(np.asarray(res_ty))
         else:
-            res_tys.append([])
+            res_tys.append(np.asarray([res_ty[0], res_ty[-1]]))
```

Comment:
Codecov patch is throwing a warning:

Codecov
src/simsopt/field/tracing.py#L173
Added line #L173 was not covered by tests

Perhaps a test with forget_exact_path==True is missing